### ISSUE TITLE
dielectricFunction oscillator method 

### ISSUE THREAD

**Issue by ivergara** [francescorandi/optics#33]
As it is now, the `dielectricFuncion()` method returns a complex value being e1 and e2.

Would it make sense to give it a flag or provide accessibility methods to return only the real or imaginary parts independently?

**Comment by francescorandi** [comment]
I was thinking about this in the past days.. it would only make sense if there was a significantly computationally faster expression to calculate only the real or only the imaginary part. But since the complex expressions are already more compact than their real and imaginary counterparts, I would use dielectricFunction().real and .imag when calling the function.

**Comment by ivergara** [comment]
I don't think that we need to focus too much on performance. I'd expect that numoy gives us enough of it by itself. Sure, it could be cool to be very efficient but most things are rather easy computationally except for the Kramer-Kronig.

My point is that maybb instead of using `dielectricFunction().imag` to implement method `e2()` calling `dielectricFunction().imag`. For sure this is in the end just cosmetics and usability.

**Comment by francescorandi** [comment]
I agree. In the last two days I realized that it is really good to have less to type ;)

**Comment by ivergara** [comment]
Then how do we close this issue?
